solrsearch-plugins/src/test/java/co.cask.hydrator.plugin/SolrSearchSinkTest.java

```diff
@@ -103,11 +103,10 @@
   private static final ArtifactRange BATCH_ARTIFACT_RANGE = new ArtifactRange(Id.Namespace.DEFAULT, "etlbatch",
                                                                               CURRENT_VERSION, true,
                                                                               CURRENT_VERSION, true);
-  private static final String SINGLE_NODE_SOLR_MODE = "Single Node Solr";
+  private static final String SINGLE_NODE_MODE = "SingleNode";
```

Comment:
you can make these fields public in the config class so that there's a single source of truth.